VSDLinearSearch.java

```diff
@@ -0,0 +1,27 @@
+/*Copyrights to vsdevelopers.io*/
+/*For more programs visit vsdevelopers.io */
+/*Java program for Linear Search*/
+public class VSDLinearSearch{

```

Comment:
remove the prefix of VSD as we changed the branding.

Reply:
just name the method as "linearSearch"

Reply:
Ok sir